contents/handbook/engineering/project-structure.md

```diff
@@ -4,7 +4,7 @@ sidebar: Docs
 showTitle: true
 ---
 
```

Comment:
Why is this link needed, and to a hard-coded commit actually? There already is an "Edit this page" button on the page